php/class-responsive-breakpoints.php

```diff
@@ -0,0 +1,336 @@
+			$palcehold_str             = Api::generate_transformation_string( $placehold_transformations );
+			$placeholder               = str_replace( $original_string, $palcehold_str, $src );
+
+			$tag_element['atts']['src'] = $placeholder;
+			if ( ! empty( $settings['lazy_preloader'] ) ) {
+				$tag_element['atts']['data-placeholder'] = $placeholder;
+			}
+		}
+
+		if ( ! empty( $settings['lazy_preloader'] ) ) {
+			$svg                              = '<svg xmlns="http://www.w3.org/2000/svg" width="' . $meta['width'] . '" height="' . $meta['height'] . '"><rect width="100%" height="100%"><animate attributeName="fill" values="rgba(200,200,200,0.3);rgba(200,200,200,1);rgba(200,200,200,0.3)" dur="2s" repeatCount="indefinite" /></rect></svg>';
+			$tag_element['atts']['src']       = 'data:image/svg+xml;utf8,' . $svg;
+			$tag_element['atts']['data-type'] = $format;
+		}
+
+		unset( $tag_element['atts']['loading'] );
+		$tag_element['atts']['decoding']   = 'async';
+		$tag_element['atts']['data-width'] = $meta['width'];
+
+		return $tag_element;
+	}
+
+	/**
+	 * Check if component is active.§
+	 *
+	 * @return bool
+	 */
+	public function is_active() {
+
+		return ! is_admin();
+	}
+
+	/**
+	 * Register assets to be used for the class.
+	 */
+	public function register_assets() {
+
+		wp_register_script( 'cld-responsive-breakpoints', $this->plugin->dir_url . 'js/responsive-breakpoints.js', null, $this->plugin->version, false );
+		if ( ! is_admin() ) {
+			$this->enqueue_assets();
+		}
+	}
+
+	/**
+	 * Enqueue Assets
+	 */
+	public function enqueue_assets() {
+
+		wp_enqueue_script( 'cld-responsive-breakpoints' );
+		$breakpoints = $this->media->get_settings()->get_value( 'image_display' );
+		wp_add_inline_script( 'cld-responsive-breakpoints', 'var CLDLB = ' . wp_json_encode( $breakpoints ), 'before' );
+	}
+
+	/**
+	 * Setup the class.
+	 */
+	public function setup() {
+
+		$this->register_settings();
+		if ( ! is_admin() ) {
+			$settings = $this->settings->get_value();
+			if ( isset( $settings['use_lazy_loading'] ) && 'on' === $settings['use_lazy_loading'] ) {
+				add_filter( 'cloudinary_pre_image_tag', array( $this, 'add_features' ), 10, 3 );
+			}
+		}
+	}
+
+	/**
+	 * Define the settings.
+	 *
+	 * @return array
+	 */
+	public function settings() {
+
+		$args = array(
+			'type'     => 'group',
+			'title'    => __( 'Lazy Loading', 'cloudinary' ),
+			'slug'     => 'lazy_loading',
+			'priority' => 9,
+			array(
+				'type'        => 'on_off',
+				'description' => __( 'Enable lazy loading', 'cloudinary' ),
+				'slug'        => 'use_lazy_loading',
+				'default'     => 'off',
+			),
+			array(
+				'type'      => 'group',
+				'condition' => array(
+					'use_lazy_loading' => true,
+				),
+				array(
+					'type'        => 'number',
+					'title'       => __( 'Lazy loading threshold', 'cloudinary' ),
+					'slug'        => 'lazy_threshold',
+					'description' => __( ' The threshold', 'cloudinary' ),
```

Comment:
@DavidCramer 

There's an space ` ` in the beginning of the string, and it's repeated a few more times.